modules/ROOT/pages/security/index.adoc

```diff
@@ -5,10 +5,12 @@ auth/authorization.adoc, auth/auth-directive.adoc, auth/subscriptions.adoc, \
 auth/authorization/allow.adoc, auth/authorization/bind.adoc, auth/authorization/roles.adoc, \
 auth/authorization/where.adoc, authentication-and-authorization/index.adoc
 
+The Neo4j GraphQL Library offers the following security features:
```

Comment:
Usually the index pages use a list with the name of the page and then a description, not incorporating the title within the description. It's not exactly a rule, but it's kinda consistent throughout docsets.